Gem/Code/Source/Components/SimplePlayerCameraComponent.cpp

```diff
@@ -6,6 +6,7 @@
  */
```

Comment:
I might be tempted to make a separate AI prefab that simply doesn't contain this component.

Reply:
Yea would be down for a change like that, it's actually how I started but the movement controller for the time being depended on it so I left it in to limit the scope of the change.